deploy/kustomize/manifests/create_namespace/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - ../marketplace
```

Comment:
Note: the base can be either `../marketplace` or `../image_registry`.
I'll add this in documentation.

Reply:
I'd suggest also to add a comment line in the yaml file (#)